addons/skin.confluence/720p/IncludesHomeRecentlyAdded.xml

```diff
@@ -140,7 +140,7 @@
 					<content>
 						<item>
 							<label>$INFO[Window.Property(LatestMovie.1.Title)]</label>
-							<label2/>
+							<label2 />
```

Comment:
Well it´s just the way the XMLWriter formats the stuff. I am sure it´s according to specifications.